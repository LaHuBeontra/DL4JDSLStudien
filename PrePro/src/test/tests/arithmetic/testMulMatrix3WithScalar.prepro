function main() {
	import mat3 testMat3_1, scal numbers;

	mat3 result = testMat3_1 * numbers;

	export result;
}
