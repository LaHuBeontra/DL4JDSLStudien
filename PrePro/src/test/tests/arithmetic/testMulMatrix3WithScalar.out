DefaultPreProDataSet{
dataSetEntries=
[DataSetEntry{variableName='result', variableShape=[4, 3, 3], startColumn=0, length=9}],
dataSet=
[[1.00,  2.00,  3.00,  4.00,  5.00,  6.00,  7.00,  8.00,  9.00],  
 [20.00,  22.00,  24.00,  26.00,  28.00,  30.00,  32.00,  34.00,  36.00],  
 [57.00,  60.00,  63.00,  66.00,  69.00,  72.00,  75.00,  78.00,  81.00],  
 [112.00,  116.00,  120.00,  124.00,  128.00,  132.00,  136.00,  140.00,  144.00]]

calculated_variables=
result=[[[1.00,  2.00,  3.00],  
  [4.00,  5.00,  6.00],  
  [7.00,  8.00,  9.00]],  

 [[20.00,  22.00,  24.00],  
  [26.00,  28.00,  30.00],  
  [32.00,  34.00,  36.00]],  

 [[57.00,  60.00,  63.00],  
  [66.00,  69.00,  72.00],  
  [75.00,  78.00,  81.00]],  

 [[112.00,  116.00,  120.00],  
  [124.00,  128.00,  132.00],  
  [136.00,  140.00,  144.00]]]
}